modules/learn/pages/data/data.adoc

```diff
@@ -24,8 +24,15 @@ Each key:
 
 * Must be a UTF-8 string with no spaces.
 Special characters, such as `(`, `%`, `/`, `"`, and `_`, are acceptable.
-* May be no longer than 250 bytes.
-* Must be unique within its bucket.
+* May be no longer than 246 bytes.
++
+[NOTE]
+====
+If a user accesses the *default* collection using a legacy SDK, then the key can be up to 250 bytes in the length.
```

Comment:
If I understood @jimwwalker 's comments correctly, it doesn't have to be because they're using a legacy SDK, that's just one example.

```suggestion
If a user accesses the *default* collection, for example by setting the query context to the default collection in the default scope, or by using a legacy SDK, then the key can be up to 250 bytes in the length.
```

Reply:
Yup, that makes sense.